types/yup/index.d.ts

```diff
@@ -89,6 +89,9 @@ export interface NumberSchemaConstructor {
 export interface NumberSchema extends Schema {
     min(limit: number | Ref, message?: string): NumberSchema;
     max(limit: number | Ref, message?: string): NumberSchema;
+    less(limit: number | Ref, message?: string): NumberSchema;
```

Comment:
It looks like it should be `lessThan`. see [number.js](https://github.com/jquense/yup/blob/v0.24.0/src/number.js)